plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -191,4 +194,17 @@ public HudiConfig setMaxOutstandingSplits(int maxOutstandingSplits)
         this.maxOutstandingSplits = maxOutstandingSplits;
         return this;
     }
+
+    @NotNull
+    public String getUserInputInstantTimeFormat()
+    {
+        return userInputInstantTimeFormat;
+    }
+
+    @Config("hudi.user-input-instant-time-format")
+    @ConfigDescription("Supported user input instantTime format for the incremental query.")
```

Comment:
I'm wondering why Hudi allows a user-defined timestamp format in the metadata column. Can you share the official documentation or link to the source code?

Reply:
removed this config